coalib/testing/LocalBearTestHelper.py

```diff
@@ -185,10 +201,11 @@ def check_results(self,
                                   create_tempfile=create_tempfile,
                                   tempfile_kwargs=tempfile_kwargs,
                                   settings=settings)
+        self.addTypeEqualityFunc(list, self.assertObjectsEqual)
```

Comment:
Another alternative that I've found is we pass the `Result` object to `assertEqual` which is something like(Error handling of empty lists, and so on will be done..) : 
```
        if not check_order:
            self.assertEqual(sorted(bear_output)[0], sorted(results)[0])
        else:
            self.assertEqual(bear_output[0], results[0])
```
Then `self.addTypeEqualityFunc(Result, self.assertObjectsEqual)` snippet is justified IIUC.